docs/Deprecating-Disabling-and-Removing-Formulae.md

```diff
@@ -65,21 +68,23 @@ The `because` parameter can be a preset reason (using a symbol) or a custom reas
 
 A formula should be removed if it does not meet our criteria for [acceptable formulae](Acceptable-Formulae.md) or [versioned formulae](Versions.md), has a non-open-source license, or has been disabled for over a year.
 
+**Note: disabled formulae in `homebrew/core` will be automatically removed one year after their disable date.**
```

Comment:
Thoughts on making this 6 months? That would bring the total deprecation + disable cycle to 1 year for popular formulae.

Reply:
Seems reasonable but: until we've update the code accordingly the docs should probably be kept correct?

Reply:
This seems like a good PR to discuss whether there's an agreement to do so however.

I perhaps pointed out the wrong line here - there's earlier lines saying that disabled formulae can't be removed even manually before the 1 year is up.

Reply:
> This seems like a good PR to discuss whether there's an agreement to do so however.

I'd love to get this PR shipped sooner rather than later once it's correct and then we can iterate further on future PRs.


> I perhaps pointed out the wrong line here - there's earlier lines saying that disabled formulae can't be removed even manually before the 1 year is up (except for unpopular formulae which is 3 months).

Could you suggest a wording change?

Reply:
> Could you suggest a wording change?

It's on unchanged lines, and GitHub doesn't support that.

But it's a `s/a year/six months/` if agreed on.